go-controller/pkg/ovn/master_test.go

```diff
@@ -1586,6 +1587,9 @@ func TestController_syncNodes(t *testing.T) {
 				sbClient,
 				record.NewFakeRecorder(0),
 				wg)
+			if err != nil {
```

Comment:
use gomega here to expect error not to occur

Reply:
This test is not written using the ginkgo framework, so to do this I had to register a gomega fail handler